Yoast/wordpress-seo
# Removed outdated dependencies.

## Summary

This PR can be summarized in the following changelog entry:

* [not-userfacing] Removed old `minimatch` dependents.

## Relevant technical choices:

* Removed grunt-phplint in favour of a grunt shell command.
* Removed grunt-wp-css because it was only being run on dist files.

## Test instructions

This PR can be tested by following these steps:

* Make sure `grunt wpcss` and `grunt phplint` commands aren't used anywhere anymore.
* Run `grunt shell:php-lint` (new command), `grunt build`, `grunt check`, `grunt css` and `grunt release`.

## UI changes

## Quality assurance


Fixes #
